Externals_CAM.cfg

```diff
@@ -2,7 +2,7 @@
 local_path = ccpp_framework
 protocol = git
 repo_url = https://github.com/peverwhee/ccpp-framework
-tag = CPF_0.2.050
+tag = CPF_0.2.053
```

Comment:
Any reason to not bump this up to `CPF_0.2.056`?

Reply:
Nope! The other PR https://github.com/ESCOMP/CAM-SIMA/pull/253 for groups bumps this up to 56, but I'm happy to bump it here (especially given the name of the PR...)